packages/server/assets/core/typedefs/apitoken.graphql

```diff
@@ -77,3 +99,12 @@ extend type Mutation {
     @hasServerRole(role: SERVER_USER)
     @hasScope(scope: "tokens:write")
 }
+
+extend type ProjectMutations {
+  createEmbedToken(token: EmbedTokenCreateInput!): String!
```

Comment:
do these need any scope checks?

Reply:
also: it would be nicer DX for the consumer if this would actually return the newly created EmbedToken entity. it probably returns the actual secret token string, which is also useful, but then you could return both

```
type CreateEmbedTokenResult {
  token: String!
  tokenMetadata: EmbedToken!
}
```

Reply:
I don't think scope checks are necessary given the policy, but I dig the return type. Done

Reply:
scope checks have a different point than auth policies - auth policies check what the user can do, scopes check what the token can do. you, as a user may be a llowed to manage embed tokens, but you still might not want to allow your token to do so